swarm.js

```diff
@@ -153,6 +156,10 @@ class Swarm extends EventEmitter {
       topic.on('update', () => this.emit('updated', { key }))
       if (lookup) {
         topic.on('peer', (peer) => {
+          if (!this.whitelist(peer)) {
+            this.emit('peer-rejected', peer, { reason: 'whitelist' })
```

Comment:
just loose the { ... } details object here. there is only a single event for this.